### PR TITLE
Deprecate OMetaDatabaseAndTable

### PR DESCRIPTION
**Pull request by MilanBariya**
### Describe your changes :

I worked on the ..... because ...
Fix: #7960
#
### Type of change :

- [x] Bug fix
- [x] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Frontend Preview (Screenshots) :
<p align="center">For frontend related change, please link screenshots of your changes preview! Optional for backend related changes.
</p>

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

File: openmetadata-docs/content/sdk/python/build-connector/source.md [hunk 1]
```diff
@@ -66,16 +66,22 @@ class SampleTablesSource(Source):
...
+    def next_record(self) -> Iterable[Entity]:
+
+        yield from self.yield_create_request_database_service(self.config)
+
+        service_entity: DatabaseService = self.metadata.get_by_name(
+            entity=DatabaseService, fqn=self.config.serviceName
+        )
+        service_id = service_entity.id
+
+        yield CreateDatabaseRequest(
```
Comment (OnkarVO7) [inline openmetadata-docs/content/sdk/python/build-connector/source.md:78]: @MilanBariya can you also add logic to ingest tables and databaseSchema

**Review by OnkarVO7 — CHANGES_REQUESTED** [review]

File: openmetadata-docs/content/sdk/python/build-connector/source.md (hunk 1 above)
Comment (MilanBariya) [reply openmetadata-docs/content/sdk/python/build-connector/source.md:78]: Done

File: openmetadata-docs/content/sdk/python/build-connector/source.md [hunk 2]
```diff
@@ -66,16 +66,49 @@ class SampleTablesSource(Source):
...
+        database_id = database_entity.id
+
+        yield CreateDatabaseSchemaRequest(
+            name="awesome-schema",
+            description="description",
+            database=EntityReference(id=database_id, type="database"),
+        )
+
+        database_schema_entity: DatabaseSchema = self.metadata.get_by_name(
+            entity=DatabaseSchema, fqn=self.config.serviceName + "." + "awesome-database" + "." + "awesome-schema"
```
Comment (OnkarVO7) [inline openmetadata-docs/content/sdk/python/build-connector/source.md:98]: @MilanBariya Instead of direct concat you can use the fqn builder here

Comment (MilanBariya) [reply openmetadata-docs/content/sdk/python/build-connector/source.md]: Done.

**Review by OnkarVO7 — DISMISSED** [review]

**Review by OnkarVO7 — APPROVED** [review]

**Pull request merged**